tests/tck/features/match/Scan.feature

```diff
@@ -155,3 +155,17 @@ Feature: Match seek by scan
       LIMIT 3
       """
     Then a ExecutionError should be raised at runtime: Scan vertices must specify limit number.
+    When executing query:
+      """
+      MATCH ()-[e:is_teacher]-()
+      RETURN type(e) AS Type, e.start_year AS StartYear, e.end_year AS EndYear
+      LIMIT 3
+      """
+    Then a ExecutionError should be raised at runtime: Scan vertices must specify limit number.
+    When executing query:
+      """
+      MATCH ()-[e]-()
+      RETURN type(e) AS Type, e.start_year AS StartYear, e.end_year AS EndYear
+      LIMIT 3
+      """
+    Then a ExecutionError should be raised at runtime: Scan vertices must specify limit number.
```

Comment:
It's confused to throw the `scan vertices` error, It's obviously `scan edge` in the query.

Reply:
This plan start from ScanVertices, and transformed to ScanEdges in optimizer, but in this case transform failed, so result in ScanVertices failed.